coalib/settings/Setting.py

```diff
@@ -135,6 +139,7 @@ def __init__(self,
         self.from_cli = from_cli
         self.key = key
         self.origin = str(origin)
+        self.to_append = to_append
```

Comment:
if `to_append` is `True` I think I shouldn't be able to retrieve the value of the setting since it's not complete